tests/kakuro/TestDBConnection.java

```diff
@@ -12,42 +16,33 @@
 
 	
 	@Test
-	public void testConnect() {
-	  
-		  	
-	  DatabaseConnection db = new DatabaseConnection();
-	  
-	  db.connect();
-	  
-	  Connection conn = db.getConnection(); 
-	  
-	  assertEquals(conn!=null, true);
-	  
-	  db.disconnect();
-	
+	public void testConnect()
+	{
```

Comment:
@changlmasaki You gotta fix the Indentation everywhere cause suddenly it goes from 4 spaces inside of a method to 1 for some reason...